Makefile.prow

```diff
@@ -58,7 +58,7 @@ envtest: $(GOBIN)/setup-envtest
 
 $(GOBIN)/setup-envtest:
 	@echo Installing envtest tools
-	GOFLAGS= go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+	GOFLAGS= go install sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.0.0-20230216140739-c98506dc3b8e
```

Comment:
This really needs to be in a separate PR. Since this PR is for velero changes that aren't (yet?) upstream, if they ever did make it upstream, we'd be dropping this commit, and then we'd lose the prow fix.

Reply:
so merge that before this?

Reply:
I think so, yes. Also, won't we need that same fix elsewhere (although maybe it's already on 1.4 and konveyor-dev). Just trying to keep carried patches from upstream code limited to just those changes.

Reply:
[will drop commit once test passes and open new pr](https://github.com/openshift/velero/pull/320)

Reply:
https://github.com/openshift/velero/pull/320

Reply:
reverted.